test/unit/migrate/Migrator.js

```diff
@@ -31,4 +36,35 @@ describe('Migrator', () => {
       }).not.to.throw();
     });
   });
+
+  describe('uses postProcessResponse for migrations', (done) => {
+    let migrationSource;
+    let knex;
+    beforeEach(() => {
+      migrationSource = new FsMigrations(
+        'test/unit/migrate/processed-migrations/'
+      );
+    });
+
+    afterEach(() => {
+      knex.destroy();
+    });
+
+    it('latest', (done) => {
+      knex = Knex({
+        ...sqliteConfig,
+        connection: ':memory:',
+        migrationSource,
+        postProcessResponse: () => {
+          done();
```

Comment:
This test actually calls done, before execution of migrations has ended. Because that is separate inmemory database, it probably won't cause any problems. Except test might have been ended already before that expectation below has finished execution.